pac4j-core/pom.xml

```diff
@@ -45,6 +45,12 @@
         </dependency>
         <dependency>
             <groupId>org.springframework</groupId>
+            <artifactId>spring-core</artifactId>
+            <scope>compile</scope>
+            <version>${spring.version}</version>
```

Comment:
It should be marked with `<optional>true</optional>`  because I don't want to pull the Spring dependency for everybody.